ttkwidgets/notebook.py

```diff
@@ -0,0 +1,940 @@
+        style.layout('Notebook.Tab.Close',
+                     [('Close.padding',
+                       {'sticky': 'nswe',
+                        'children': [('Close.border',
+                                      {'border': '1',
+                                       'sticky': 'nsew',
+                                       'children': [('Close.close',
+                                                     {'sticky': 'ewsn'})]})]})])
+        style.layout('Left.Notebook.TButton',
+                     [('Button.padding',
+                       {'sticky': 'nswe',
+                        'children': [('Button.leftarrow', {'sticky': 'nswe'})]})])
+        style.layout('Right.Notebook.TButton',
+                     [('Button.padding',
+                       {'sticky': 'nswe',
+                        'children': [('Button.rightarrow', {'sticky': 'nswe'})]})])
+        style.configure('Notebook', **style_config)
+        style.configure('Notebook.Tab', relief='raised', borderwidth=1,
+                        **style_config)
+        style.configure('Notebook.Tab.Frame', relief='flat', borderwidth=0,
+                        **style_config)
+        style.configure('Notebook.Tab.Label', relief='flat', borderwidth=1,
+                        padding=0, **style_config)
+        style.configure('Notebook.Tab.Label', foreground=theme['unselectedfg'])
+        style.configure('Notebook.Tab.Close', relief='flat', borderwidth=1,
+                        padding=0, **style_config)
+        style.configure('Notebook.Tab.Frame', background=theme['bg'])
+        style.configure('Notebook.Tab.Label', background=theme['bg'])
+        style.configure('Notebook.Tab.Close', background=theme['bg'])
+
+        style.map('Notebook.Tab.Frame',
+                  **{'background': [('selected', '!disabled', theme['activebg'])]})
+        style.map('Notebook.Tab.Label',
+                  **{'background': [('selected', '!disabled', theme['activebg'])],
+                     'foreground': [('selected', '!disabled', theme['fg'])]})
+        style.map('Notebook.Tab.Close',
+                  **{'background': [('selected', theme['activebg']),
+                                    ('pressed', theme['darkcolor']),
+                                    ('active', theme['activebg'])],
+                     'relief': [('hover', '!disabled', 'raised'),
+                                ('active', '!disabled', 'raised'),
+                                ('pressed', '!disabled', 'sunken')],
+                     'lightcolor': [('pressed', theme['darkcolor'])],
+                     'darkcolor': [('pressed', theme['lightcolor'])]})
+        style.map('Notebook.Tab',
+                  **{'ba.ckground': [('selected', '!disabled', theme['activebg'])]})
+
+        style.configure('Left.Notebook.TButton', padding=0)
+        style.configure('Right.Notebook.TButton', padding=0)
+
+        style.configure('TNotebook.Tab', background=theme['bg'],
```

Comment:
I am not sure to understand why you change the `ttk.Notebook` theme here.

Reply:
To be fair, I pretty much copied and pasted your styling code verbatim, only integrating it into the widget itself. I think these can be removed

Reply:
I think there was a misunderstanding, I should have given the line numbers: I was speaking about the `style.configure('TNotebook.Tab', ...` and `style.map('TNotebook.Tab', ...` (lines 456-460) which are changing the `TNotebook` style. The lines you removed where configuring the notebook's tab scrolling buttons and if you run the example you will see that their appearance has changed.

Reply:
I've addressed this in my latest commit